engine/runtime-integration-tests/src/test/resources/Test_Fully_Qualified_Name_Conflict/src/Main.enso

```diff
@@ -4,4 +4,5 @@ from Standard.Base.Errors.Common import all
 
 main =
     Atom.A 1
+    # `local.Test_Fully_Qualified_Name_Conflict.Atom` should be resolved to a Type, not to a module
```

Comment:
That one is a bit surprising

Reply:
The code was inherited from #9367, CCing @Akirathan